### DISCUSSION TITLE
Convert GEDCOM 5.5.1 to GEDCOM 7

### DISCUSSION THREAD

**Discussion by HainesB** [FamilySearch/GEDCOM#237] [Q&A] [answered]
I pick up that there's a way to convert a GEDCOM 5.z file to 7.x, but my searching did not stumble on it. Would some kind soul point me in the right direction? Also some general advice would be nmuch appreciated.

**Comment by tychonievich** [comment, marked as answer]
<https://gedcom.io/tools/> contains several tools, including 5.5.1-to-7.0.0 converters

**Comment by HainesB** [comment]
On Tue, Nov 01, 2022 at 12:37:32PM -0700, Luther Tychonievich wrote:
> <https://gedcom.io/tools/> contains several tools, including 5.5.1-to-7.0.0 converters

The pointer much apprciated. I had already taken a look at the C 
concerter and didn''t know that to do with it. I'Ve comiled C cource 
fdiles from GiHub more. But in this case the Makefile file was htmtl, 

I made a regular file with its content, but I don't know how to change 
first line (cc: = clang... ) to point to my compiler (gcc)

I know this kind of problem is irrelevant to the blog, but toss it in 
just so it reflects that some folks do not know C progfamming

I am inclined to avoid the online converted for reasons of security.

-- 

     Haines Brown 
 /"\
 \ /  ASCII Ribbon Campaign
  X    against HTML e-mail 
 / \
